buildSrc/src/main/java/Dependencies.kt

```diff
@@ -0,0 +1,50 @@
+object Versions {
+    const val ANDROID_COMPILE_SDK_VERSION = 33
+    const val ANDROID_MIN_SDK_VERSION = 24
+    const val ANDROID_TARGET_SDK_VERSION = 33
+
+    const val ANDROID_VERSION_CODE = 1
+    const val ANDROID_VERSION_NAME = "0.1.0"
+
+    const val ANDROIDX_ACTIVITY_COMPOSE = "1.5.1"
+
+    const val COMPOSE = "1.3.0"
+    const val COMPOSE_NAVIGATION = "2.5.2"
+
+    const val GRADLE = "7.0.4"
+
+    const val JUNIT = "4.13.2"
+
+    const val KOTLIN = "1.7.10"
+    const val KOTLIN_COROUTINES = "1.6.4"
+
+    const val TIMBER = "4.7.1"
+}
+
+object Deps {
```

Comment:
Is this the default Android way to naming for `Dependencies`? I think it's better to name with full word `Dependencies` to avoid confusion like [this one](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Dependencies.kt).

Reply:
@blyscuit It's not a default way of writing, it's shorter imo 😄 Let me switch to use the full name `Dependencies`, no worries 👌

Reply:
@blyscuit Fixed in https://github.com/luongvo/kmm-survey/pull/40/commits/c91e100e33cc40266599e02c9cc7874378e90a9f